src/Raven.Server/Smuggler/Documents/Data/SmugglerResult.cs

```diff
@@ -21,7 +21,8 @@ public SmugglerResult()
 
             Documents = new CountsWithSkippedCountAndLastEtag();
             RevisionDocuments = new CountsWithLastEtag();
-            Identities = new Counts();
+            LocalIdentities = new Counts();
```

Comment:
Studio is not a problem, it needs to be adjusted anyway, question is what impact does it have for 3.x <=> 4.0 import/export?